src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheck.java

```diff
@@ -72,6 +78,12 @@
      */
     private boolean ignoreOverridden;
 
+    /** The visibility scope where methods are checked. */
+    private Scope scope = Scope.ANONINNER;
```

Comment:
Kept code as it is.  I can't remove this reference to ANONINNER and keep the code as it is because if e.g. I choose PRIVATE as the default scope, then anonymous inner functions are not checked. To avoid using this constant I could use null or NONE with the drawbacks I described a few comments above